discoverd/client/client.go

```diff
@@ -23,6 +24,9 @@ var ErrTimedOut = errors.New("discoverd: timed out waiting for instances")
 
 var defaultLogger = log15.New("component", "discoverd")
 
+var RequestDeadline = 10 * time.Second
+var RetryInterval = 100 * time.Millisecond
```

Comment:
Should these be constants or exposed publicly? Normally exposing global variables from a package is a bad idea. If you want them to be configurable, they should be stored as client configuration.